contracts/error/Errors.sol

```diff
@@ -140,6 +140,8 @@ library Errors {
 
     // FeeHandler errors
     error InvalidClaimFeesInput(uint256 marketsLength, uint256 tokensLength);
+    error InvalidSwapTokenInput(address swapToken, uint256 arrayPosition);
```

Comment:
can we call the token "buybackToken" and call the error "InvalidBuybackTokenInput"? i think that would be clearer to differentiate it from the regular GMX V2 swaps

Reply:
I've updated accordingly in the latest commit